### ISSUE TITLE
Unhandled exception executing tab-completer

### ISSUE THREAD

**Issue by andris155** [utarwyn/EnderContainers#15]
```
[23:03:29] [Server thread/ERROR]: Exception when Huntermaster attempted to tab complete ecp backup
org.bukkit.command.CommandException: Unhandled exception executing tab-completer for 'ecp backup' in fr.utarwyn.endercontainers.commands.MainCommand(endercontainers)
	at org.bukkit.command.SimpleCommandMap.tabComplete(SimpleCommandMap.java:236) ~[patched_1.13.1.jar:git-Paper-"a3677899"]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.tabCompleteCommand(CraftServer.java:1742) ~[patched_1.13.1.jar:git-Paper-"a3677899"]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.tabComplete(CraftServer.java:1714) ~[patched_1.13.1.jar:git-Paper-"a3677899"]
	at org.bukkit.craftbukkit.v1_13_R2.command.BukkitCommandWrapper.getSuggestions(BukkitCommandWrapper.java:49) ~[patched_1.13.1.jar:git-Paper-"a3677899"]
	at com.mojang.brigadier.tree.ArgumentCommandNode.listSuggestions(ArgumentCommandNode.java:68) ~[patched_1.13.1.jar:git-Paper-"a3677899"]
	at com.mojang.brigadier.CommandDispatcher.getCompletionSuggestions(CommandDispatcher.java:379) ~[patched_1.13.1.jar:git-Paper-"a3677899"]
	at net.minecraft.server.v1_13_R2.PlayerConnection.sendSuggestions(PlayerConnection.java:568) ~[patched_1.13.1.jar:git-Paper-"a3677899"]
	at net.minecraft.server.v1_13_R2.PlayerConnection.lambda$a$2(PlayerConnection.java:554) ~[patched_1.13.1.jar:git-Paper-"a3677899"]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_181]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_181]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SourceFile:199) ~[patched_1.13.1.jar:git-Paper-"a3677899"]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1011) ~[patched_1.13.1.jar:git-Paper-"a3677899"]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:428) ~[patched_1.13.1.jar:git-Paper-"a3677899"]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:938) ~[patched_1.13.1.jar:git-Paper-"a3677899"]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:836) ~[patched_1.13.1.jar:git-Paper-"a3677899"]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
Caused by: java.lang.ArrayIndexOutOfBoundsException: -1
	at fr.utarwyn.endercontainers.commands.AbstractCommand.tabComplete(AbstractCommand.java:148) ~[?:?]
	at fr.utarwyn.endercontainers.commands.AbstractCommand.tabComplete(AbstractCommand.java:144) ~[?:?]
	at org.bukkit.command.Command.tabComplete(Command.java:90) ~[patched_1.13.1.jar:git-Paper-"a3677899"]
	at org.bukkit.command.SimpleCommandMap.tabComplete(SimpleCommandMap.java:230) ~[patched_1.13.1.jar:git-Paper-"a3677899"]
	... 15 more
```

**Comment by utarwyn** [comment]
I will investigate to fix this error.
Thank you!

**Comment by utarwyn** [comment]
This error has been fixed in the following commit: f67aad4514e52135237634cc94e9792f14f09a22
I think that this error only occurs on 1.13 versions that is not fully supported by **EnderContainers**. I will work to make it work on that version soon.
